src/test/java/com/neo4j/docker/TestPluginInstallation.java

```diff
@@ -0,0 +1,140 @@
+package com.neo4j.docker;
+
+import com.neo4j.docker.utils.ExampleNeo4jPlugin;
+import com.neo4j.docker.utils.HostFileHttpHandler;
+import com.neo4j.docker.utils.HttpServerRule;
+import com.neo4j.docker.utils.JarBuilder;
+import com.neo4j.docker.utils.SetContainerUser;
+import com.neo4j.docker.utils.TestSettings;
+import org.junit.Rule;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
+import org.junit.rules.TemporaryFolder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.Testcontainers;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.shaded.com.google.common.io.Files;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import org.neo4j.driver.AuthTokens;
+import org.neo4j.driver.Driver;
+import org.neo4j.driver.GraphDatabase;
+import org.neo4j.driver.Record;
+import org.neo4j.driver.Session;
+import org.neo4j.driver.StatementResult;
+
+import static com.neo4j.docker.utils.TestSettings.NEO4J_VERSION;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@EnableRuleMigrationSupport
+public class TestPluginInstallation
+{
+    private static final int DEFAULT_BROWSER_PORT = 7474;
+    private static final int DEFAULT_BOLT_PORT = 7687;
+
+    private static final String versions = "/versions.json";
+    private static final String myPlugin = "/myPlugin.jar";
+
+    private static final Logger log = LoggerFactory.getLogger( TestPluginInstallation.class );
+
+    @Rule
```

Comment:
In JUnit5 TemporaryFolder is replaced by:
https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory